seed/challenge-style-guide.md

```diff
@@ -63,6 +63,39 @@ Here are some example challenge names:
 - Condense arrays with .reduce
 - Use Bracket Notation to Find the First Character in a String
 
+## Numbering Challenges
+
+Every challenge needs an `id`. If you don't specify one, then MongoDB will create a new random one when it saves the data; however, we don't want it to do that, since we want the challenge ids to be consistent across different environments (staging, production, lots of different developers, etc.).
+
+To generate a new one in a shell (assuming MongoDB is running separately):
+
+1. Run `mongo` command
+2. Run `ObjectId()` command
+
+For example:
+
+```
+$ mongo
+MongoDB shell version v3.6.1
+connecting to: mongodb://127.0.0.1:27017
+MongoDB server version: 3.4.10
+...
+$ ObjectId()
+ObjectId("5a474d78df58bafeb3535d34")
+```
+
+The result is a new id, for example `5a474d78df58bafeb3535d34` above.
+
+Once you have your id, put it into the JSON file as the `id` field, e.g.
+
+```json
```

Comment:
` ```json ` `=>` ` ``` `

I know, github is weird.

Reply:
my editor (RubyMine) displays here-doc text with styling if you provide it a language hint as above, and automatically inserts it; I'm happy to delete the hint, but I don't think GitHub cares one way or the other, does it?

Reply:
![image](https://user-images.githubusercontent.com/8524/35160333-d23431ea-fd0a-11e7-9cef-9c3af7baa7dd.png)
this screengrab shows it's ignoring the `json` after the triple-backtick -- again, I'm happy to delete it if you think it's confusing to future documentarians

Reply:
OHHH, it's complaining about the comment in the other file's JSON snippet. Which makes sense since JSON doesn't support comments.

Reply:
aha, https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting -- I now think this is a super-cool feature and recommend keeping it